overlord/servicestate/servicestate.go

```diff
@@ -228,6 +228,11 @@ func (sd *StatusDecorator) DecorateWithStatus(appInfo *client.AppInfo, snapApp *
 		// nothing to do
 		return nil
 	}
+	if snapApp.DaemonScope != snap.SystemDaemon {
```

Comment:
Bit of tangent and probably a separate refactor would be needed since StatusDecorator always uses system instance of snapd, but we could at least show the global user enable/disable state.

Reply:
I tried to keep this branch as targeted as possible.  However, the snap (which is published on the store rather than assembled with `snap pack`) I used in the spread test included both a system and session bus service.  This was the minimal change for `snap services` to not fall over.

There will be a follow up addressing user daemons.